README.md

```diff
@@ -11,7 +11,33 @@ If you're interested in the research network, see our [pages](https://network-di
 more information. Or get in touch with one of the network members.
 
 
-## How to Contribute to the Pages
+## How to Contribute to the Pages (Step by Step)
+
+The basis of the pages are markup-files. If you want to understand how markup files are rendered as a web page, see the technical explanation below.
+This section will give you brief explanation on how to edit what is visible to the public in a way that is almost as easy as writing in the text editor of your choice.
+
+### Step One
+
+The most important thing is to first create a branch in GitHub for your edit. This makes sure there is some form of editorial oversight and changes
+can be reviewed before being published. You can find out how to create a branch [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository)
+
+When creating a branch, give it a name like this: "wip/YOUR_BRANCH_NAME".

```

Comment:
thinking about it: maybe we don't even need `wip/`. There won't ever be any non work-in-progress branches here, except main. What do you think?